app/courses/article/[slug]/page.tsx

```diff
@@ -12,3 +12,55 @@ const Article = ({ params }: Props) => {
 };
 
 export default Article;
+
+// "use client"
```

Comment:
GIt will hold onto this stuff, we try not to push commented code normally, unless there's a really good reason to.

Reply:
good to know - this is just some chatGPT generated code that I went in cricles on trying to learn, so is just a reminder for me to talk to you about how it works / should be written

Reply:
ChatGPT is great for some tasks and terrible for other. Personally, I think it sucks at react. It might write code that runs, but that is a very low bar.

Reply:
Yeah its been great for styling, Im defintely not trying to rely on it for things I dont know - just getting it to have a go, see if I can use that to piece things together. In this instance it was a bit beyond